controllers/users.js

```diff
@@ -22,14 +22,14 @@ router.get('/:id/spots', function(req, res) {
 // Create a new user
 router.post('/', function(req, res) {
   if (!req.body.hasOwnProperty('email') ||
-      !req.body.hasOwnProperty('password') ||
+      !req.body.hasOwnProperty('uid') ||
         !req.body.hasOwnProperty('firstName') ||
           !req.body.hasOwnProperty('lastName')) {
     res.status(400).json({error: "Missing expected body parameter."});
     return;
   }
 
-  User.create(req.body.email, req.body.password, req.body.firstName,
+  User.create(req.body.email, req.body.uid, req.body.firstName,
```

Comment:
So this means everytime we want to return the email address from the server we'd need to make another request to `authWithPassword()` and return the email from that, instead of just storing it with the actual /user data